contents/testing/writing-integration-test.md

```diff
@@ -0,0 +1,143 @@
+<frontmatter>
+  title: Integration Testing
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Integration Testing
+
+**Author(s): [Jacob Li PengCheng](https://github.com/jacoblipech)**
+
+Reviewers:
+
+## What is integration testing?
+
+Integration testing is a part of software testing where individual units of your application are combined and tested as a group. This usually happens after [unit testing stage](http://softwaretestingfundamentals.com/unit-testing/) and before the [system testing stage](https://www.guru99.com/system-testing.html).
+
+## Why is integration testing important to SE?
```

Comment:
Not sure if this is just me, but I feel like SE should be changed to software engineering 😅